ios/RNCWebView.m

```diff
@@ -1080,7 +1080,9 @@ - (void)                  webView:(WKWebView *)webView
   if (_onLoadingStart) {
     // We have this check to filter out iframe requests and whatnot
     BOOL isTopFrame = [request.URL isEqual:request.mainDocumentURL];
-    if (isTopFrame) {
+    // Do not notify event if the request is for new window
+    BOOL isMainFrame = navigationAction.targetFrame.isMainFrame;
+    if (isTopFrame && isMainFrame) {
```

Comment:
@kazkondo When user presses a link that will create a new window, `targetFrame.isMainFrame` is FALSE, right?
The current logic has a bug that notify event `onLoadingStart` even with new window case => url of the new window is updated to current parent window

Reply:
I'm not sure about the targetFrame.isMainFrame specs, so please test carefully